src/messages/manifestjson.js

```diff
@@ -455,6 +455,15 @@ export const WRONG_ICON_EXTENSION = {
   file: MANIFEST_JSON,
 };
 
+export const DUPLICATE_GECKO_PROPERTY = {
+  code: 'DUPLICATE_GECKO_PROPERTY',
+  message: i18n._('Multiple use of "Gecko" property'),
+  description: i18n._(
+    oneLine`the property Gecko is being used in "applications" and "browser_specific_settings" keys at the same time`
```

Comment:
@wagnerand do you think that it may be worth as part of this new message to also suggest to the developer to remove `"applications"` and prefer to use just `"browser_specific_settings"`?

Reply:
@rpl Yes, maybe something like:

_The `applications` key is being ignored because it is superseded by the `browser_specific_settings` key that is also defined in the manifest. Consider removing `applications`_

Reply:
@Yrkan let's use the message suggested by Andreas in https://github.com/mozilla/addons-linter/pull/3425#discussion_r513417669 in the next updates to this PR.

Reply:
And @rpl reminded me that `property` is the proper word for `key`, so it would be:

_The `applications` property is being ignored because it is superseded by the `browser_specific_settings` property which is also defined in your manifest. Consider removing `applications`_.